.planning/codebase/TESTING.md

```
@@ -0,0 +1,562 @@
+    {
+        claims.Add(new Claim("permission", permission));
+    }
+
+    var identity = new ClaimsIdentity(claims, "TestAuth");
+    return new ClaimsPrincipal(identity);
+}
+```
+
+---
+
+*Testing analysis: 2026-01-19*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)